test/models/product_test.rb

```diff
@@ -78,4 +78,37 @@
       expect(result).must_equal false
     end
   end
+  
+  describe "relationships" do
+    before do
+      @user = User.create(username: "test user", email: "test email") 
+      @category1 = Category.create(name: "flower")
+      @category2 = Category.create(name: "specialty")
+      @product = Product.create(user_id: @user.id, name: "test product", description: "cool product", price: 1.9, photo_url: "url", stock: 3, categories: [@category1, @category2]) 
+      
+      @review1 = Review.create(product_id: @product.id, rating: 5, comment: "great product")
+      @review2 = Review.create(product_id: @product.id, rating: 2, comment: "bad product")
+      
+      @order = Order.create(email: "test email", address: "test address", name: "test user name", cc_num: 1234, zip: 98111)
```

Comment:
Same thing about the email
```suggestion
      @order = Order.create(email: "test-email@example.com", address: "test address", name: "test user name", cc_num: 1234, zip: 98111)
```